### ISSUE TITLE
Request for changes

### ISSUE THREAD

**Issue by r-nikhil** [Shreshth-mehra/Obsidian-TV-Tracker#3]
Hey, I know this is a free plugin but I have some small UI fixes which I would love for it see fixed. Hope you can work on this on your free time. 
- Add a drop down for "status" inside the "add new" button. Prefill the items (progerss, watched, watchlist etc)
- Give a scroller/dropdown for the rating input box. Its just between 10 values and input box is overkill. maybe a slider? 
- Move discover to top right/separate section inside the note. Its a totally separate feature and its better to keep it separate on the side. 
- The showcase filters have very poor alignment. Maybe move them around (on top of the showcase). Legend/Geek numbers etc can be moved out to the right or remove honestly. Zeroing out all values in the settings(for the geeknumbers) still shows an empty dropdown. A toggle to hide both would be great. 
- Make search center and prominent. 

Basically bunch of UI changes. Don't have the skills to do the PR myself but would love it if you can look into this if free

**Comment by Shreshth-mehra** [comment]
Thank you for the feedback. I'll see if I can add some customization to the plugin to allow you to make some of these changes for yourself. I have a busy couple of days but I should be able to do this in a week or 2. Once again, thanks for the feedback and you can keep the suggestions coming, happy to implement them.

**Comment by Shreshth-mehra** [comment]
I have made a new release (1.2.0) with some changes I could make quickly.

1. Added settings to hide the Legend and Metrics
2. Replaced the Rating input with a slider

The other requested changes require a bit more thought and investigation. I'll try to fix them in the next release
